server/server.go

```diff
@@ -1351,6 +1345,11 @@ func RunServerWithOpts(stanOpts *Options, natsOpts *server.Options) (newServer *
 		nOpts = natsOpts.Clone()
 	}
 
+	// Default cluster Raft log path to ./<cluster-id>/<node-id> if not set.
+	if sOpts.Clustering.RaftLogPath == "" {
```

Comment:
Move this down where you already test for isClustered() (to do the file store sync override), or move the file store override back up here. Point is, this should be done only in clustered mode.